fyle/platform/internals/list_resources.py

```diff
@@ -23,7 +23,23 @@ def list(self, query_params=None) -> Dict:
         :return: List of Resource Objects
         """
         query_params = {} if query_params is None else query_params
-        return self.api.make_get_request(
-            api_url=self.endpoint,
-            query_params=query_params
-        )
+        if query_params.get('offset') or query_params.get('limit'):
```

Comment:
can you make offset and limit mandatory..

have a separate list_all call that doesn't take offset and limit